File: py-codellama7B-AWQ-all/taskid-11_string_xor-gen9.py
def string_xor(a: str, b: str) -> str:
    """ Input are two strings a and b consisting only of 1s and 0s.
    Perform binary XOR on these inputs and return result also as a string.
    >>> string_xor('010', '110')
    '100'
    """

    assert len(a) == len(b)
    result = ''
    for (char1, char2) in zip(a, b):
        result += str(int(char1) ^ int(char2))
    return result

File: java-codellama7B-AWQ-all/taskid-11_string_xor-gen9.java
import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    // Input are two strings a and b consisting only of 1s and 0s.
    // Perform binary XOR on these inputs and return result also as a string.
    // >>> stringXor(("010"), ("110"))
    // ("100")
    public static String stringXor(String a, String b) {

        char[] array1 = a.toCharArray();
        char[] array2 = b.toCharArray();
        StringBuilder result = new StringBuilder(array1.length);
        for (int i = 0; i < array1.length; i++) {
            if (array1[i] == array2[i]) {
                result.append("0");
            } else {
                result.append("1");
            }
        }
        return result.toString();
    }
    public static void main(String[] args) {
    assert(stringXor(("111000"), ("101010")).equals(("010010")));
    assert(stringXor(("1"), ("1")).equals(("0")));
    assert(stringXor(("0101"), ("0000")).equals(("0101")));
    }

}
